src/lib/operators/abstract_aggregate_operator.cpp

```diff
@@ -29,24 +29,24 @@ const std::vector<std::shared_ptr<AggregateExpression>>& AbstractAggregateOperat
 const std::vector<ColumnID>& AbstractAggregateOperator::groupby_column_ids() const { return _groupby_column_ids; }
 
 std::string AbstractAggregateOperator::description(DescriptionMode description_mode) const {
-  const auto* const separator = description_mode == DescriptionMode::SingleLine ? " " : "\n";
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
 
   std::stringstream desc;
-  desc << AbstractOperator::description(description_mode) << separator << "GroupBy ColumnIDs: ";
+  desc << AbstractOperator::description(description_mode) << separator;
+  desc << "GroupBy ColumnIDs: {";
```

Comment:
Should we use `[` over `{`? I associate `{` with sets and `[` with lists.

Reply:
Good point 👍

Reply:
Although GroupBy-IDs should be a set?